easybuild/tools/filetools.py

```diff
@@ -513,14 +513,15 @@ def download_file(filename, url, path, forced=False):
     # use custom HTTP header
     headers = {'User-Agent': 'EasyBuild', 'Accept': '*/*'}
     # for backward compatibility, and to avoid relying on 3rd party Python library 'requests'
-    url_req = urllib2.Request(url, headers=headers)
-    used_urllib = urllib2
+    url_req = std_urllib.Request(url, headers=headers)
+    used_urllib = std_urllib
 
     while not downloaded and attempt_cnt < max_attempts:
         try:
-            if used_urllib is urllib2:
-                # urllib2 does the right thing for http proxy setups, urllib does not!
-                url_fd = urllib2.urlopen(url_req, timeout=timeout)
+            if used_urllib is std_urllib:
+                # urllib2 (Python 2) / urllib.request (Python 3) does the right thing for http proxy setups,
+                # urllib does not!
```

Comment:
is this still relevant? Do we use urllib anywhere?

Reply:
No, but we used to, and I guess the comment serves as a reminder why we use `urllib2` rather than `urllib`, doesn't hurt to keep it there imho.